src/main/java/ru/yandex/practicum/filmorate/exception/ErrorHandler.java

```diff
@@ -0,0 +1,62 @@
+package ru.yandex.practicum.filmorate.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestControllerAdvice
+public class ErrorHandler {
+    private static final String VALIDATION_ERROR = "Ошибка валидации";
+    private static final String NOT_FOUND_ERROR = "Объект не найден";
+    private static final String SERVER_ERROR = "Внутренняя ошибка сервера";
+
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public Map<String, Object> handleValidationException(ValidationException ex) {
```

Comment:
Давай избавимся от сокращений в названиях параметров:
```suggestion
    public Map<String, Object> handleValidationException(ValidationException exception) {
```

Reply:
ушел от сокращений, я правильно понимаю сокращения мы не используем только потому чтобы код лучше воспринимался на глаз, тому кто будет его читать кроме самого разработчика? Или это какое то строгое правило? Или же зависит от устоев внутри команды?